docs/user_guide/algorithm_list.rst

```diff
@@ -225,3 +305,21 @@ These algorithms are small utility functions which perform common operations nee
 
     Collapse a Graph into a smaller Graph by combining clusters of nodes into a single node.
     ``labels`` indicates the node groupings. ``aggregator`` indicates how to combine edge weights.
+
+.. py:function:: util.graph.isomorphic(g1: Graph, g2: Graph) -> bool
+
+    Indicates whether ``g1`` and ``g2`` are isomorphic.
+
+.. py:function:: util.node_embedding.apply(embedding: NodeEmbedding, nodes: Vector) -> Matrix
+
+    Returns a dense matrix given an embedding and a vector of NodeIDs.
+
+
+Embedding
+---------
+
+Embeddings convert graph nodes or whole graphs into a dense vector representations.
+
+.. py:function:: embedding.train.node2vec(graph: Graph, p: float, q: float, walks_per_node: int, walk_length: int, embedding_size: int, epochs: int, learning_rate: float) -> NodeEmbedding
+
+    Computes the node2vec embedding.
```

Comment:
I think it would be useful to link the paper somehow.

We could link the [SNAP page](https://snap.stanford.edu/node2vec/) or the [arxiv page](https://arxiv.org/abs/1607.00653) or [the paper directly from arxiv](https://arxiv.org/pdf/1607.00653.pdf).

I have a preference for citing the SNAP page.

Reply:
Added a link to the SNAP page